micrometer-jetty11/build.gradle

```diff
@@ -2,11 +2,7 @@ description 'Micrometer instrumentation for Jetty 11'
 
 dependencies {
     api project(":micrometer-core")
-    api(libs.jetty11Server) {
-        version {
-            strictly libs.jetty11Server.get().version
```

Comment:
This was no longer needed since I removed the jetty dependencies from `dependencies.gradle` in lieu of referring to the type-safe version catalog everywhere for jetty.